[xharness] Bump timeout for .NET tests again.

rolfbjarne opened the pull request:
We keep adding more and more test cases...

mandel-macaque reviewed (approved):
yikes...

vs-mobiletools-engineering-service2 commented:
### :x: [PR Build] Tests failed on Build :x:

Tests failed on Build. 
* [Azure DevOps](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=5104451&view=ms.vss-test-web.test-result-details)
* [Html Report (VSDrops)](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/sim/;/tests/vsdrops_index.html)
# API diff
:white_check_mark: API Diff from stable
<details><summary>View API diff</summary>

* iOS [vsdrops](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/apidiff/;/ios-api-diff.html) [gist](https://gist.github.com/c1b637b278121688182cde6d321e62c0)
* macOS [vsdrops](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/apidiff/;/mac-api-diff.html) [gist](https://gist.github.com/eddc1057ffd36deeceaef0c0c767599f)
* macCat [vsdrops](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/apidiff/;/maccat-api-diff.html) [gist](https://gist.github.com/220722d6ad9cc861780e92d498ddb4b4)
* macCatiOS  [gist](https://gist.github.com/dec19792953a772386cb14dc752c1f98)
* tvOS [vsdrops](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/apidiff/;/tvos-api-diff.html) [gist](https://gist.github.com/2424599b54e2189acecc67d0694e46a1)
* watchOS [vsdrops](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/apidiff/;/watchos-api-diff.html) [gist](https://gist.github.com/79f747c8fd539ea8b39a11a31c06e5bf)
</details>

# API & Generator diff

:white_check_mark: [API Diff (from PR only)](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/apigeneratordiff/;//api-diff.html) (no change)
:white_check_mark: [Generator Diff](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210818.5/5104451/apigeneratordiff/;//generator-diff/index.html) (only version changes)

# GitHub pages

Results can be found in the following github pages (it might take some time to publish):

* [Test results](https://xamarin.github.io/macios.ci/pr/PR12470/5104451/HtmlReport-sim/tests/vsdrops_index.html)
* [API diff ](https://xamarin.github.io/macios.ci/pr/PR12470/5104451/HtmlReport-sim/api-diff/api-diff.html)
* [API & Generator diff](https://xamarin.github.io/macios.ci/pr/PR12470/5104451/apicomparison/api-diff.html)


# Test results

<details>
<summary>3 tests failed, 144 tests passed.</summary>

## Failed tests

 * link all/Mac Catalyst/Debug [dotnet]: TimedOut (Execution timed out after 1200 seconds.
No test log file was produced)
 * link sdk/Mac [dotnet]/Debug [dotnet]: BuildFailure
 * link sdk/Mac [dotnet]/Release [dotnet]: BuildFailure
</details>


[Pipeline](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=5104451&view=ms.vss-test-web.test-result-details) on Agent XAMBOT-1100.BigSur'
Merge 1c589e15e8922825ae011f419873f6651b4d367f into ef61ab51f6df015c21471e9c339b73f121d63195

rolfbjarne commented:
Test failures are unrelated:

* link all/Mac Catalyst/Debug [dotnet]: https://github.com/xamarin/maccore/issues/2414
* link sdk/Mac [dotnet]/Debug [dotnet]: Fixed in #12475.
* link sdk/Mac [dotnet]/Release [dotnet]: Fixed in #12475.
